experiments/tf_trainer/common/text_preprocessor.py

```diff
@@ -116,15 +120,46 @@ def create_estimator_with_embedding(
 
     Returns:
       TF Estimator with embedding ops added.
+
+    Note: We need to consider the case of large embeddings (see: 
+      https://stackoverflow.com/questions/48217599/
+      how-to-initialize-embeddings-layer-within-estimator-api/48243086#48243086).
+
     """
     old_model_fn = estimator.model_fn
     old_config = estimator.config
     old_params = estimator.params
 
+    def add_init_fn_to_estimatorSpec(estimator_spec, init_fn):
```

Comment:
Thanks for writing this, but wow it seems complex. Why is this necessary?

Reply:
From our discussion, should I add information in here?